protocol/contracts/libraries/Silo/LibSilo.sol

```diff
@@ -510,19 +510,25 @@ library LibSilo {
         }
         // If a Sop has occured since last update, calculate rewards and set last Sop.
         if (s.season.lastSopSeason > lastUpdate) {
-            s.a[account].sop.plenty = balanceOfPlenty(account);
+            address[] memory tokens = LibWhitelistedTokens.getWhitelistedWellLpTokens();
```

Comment:
Same reminder of the previous comment - assumption that all whitelisted well tokens are soppable

Reply:
Now have functions to store the currently-soppable wells as well as wells that were at once point soppable https://github.com/BeanstalkFarms/Beanstalk/commit/ae19fe2cb913a9473ee34ab22f54533aa6a237e6#diff-4a3dfd414e5a3956a672ce816084fc50d637eb324c8270c7856ff8fe2b71009fR153